control/pure_pursuit/include/pure_pursuit/pure_pursuit_lateral_controller.hpp

```diff
@@ -105,6 +111,10 @@ class PurePursuitLateralController : public LateralControllerBase
   autoware_auto_vehicle_msgs::msg::SteeringReport::ConstSharedPtr current_steering_;
   boost::optional<AckermannLateralCommand> prev_cmd_;
 
+  // Debug Publisher
+  rclcpp::Publisher<visualization_msgs::msg::MarkerArray>::SharedPtr pub_debug_marker_;
+  rclcpp::Publisher<autoware_auto_system_msgs::msg::Float32MultiArrayDiagnostic>::SharedPtr
```

Comment:
The `autoware_auto_system_msgs::msg::Float32MultiArrayDiagnostic` depends on the deprecated msg type (std_msgs) in ros2. Recently we removed the Float32MultiArrayDiagnostic dependency from the control module, so would you replace Float32MultiArrayDiagnostic with autoware_debug_msg as in  https://github.com/autowarefoundation/autoware.universe/pull/2225 ?